web-marketplace/lingui.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { LinguiConfig } from '@lingui/conf';
```

Comment:
could the config be similar for web-components? should we have one root config file and then config files in each packages extended from the root one? ref: https://lingui.dev/guides/monorepo
The thing is that web-components is also used by web-www that we doesn't need to be multilingual (at least for now, as part of this project)

Reply:
Sure, I'll update that.

Reply:
I did not add a `lingui.config.ts` to `web-components` though, as this package should probably not be concerned with i18n. If there are hard-coded strings in `web-components`, we might want to extract them.

Reply:
ok yeah makes sense, we do have some hard coded strings in there
so the idea would be to have them set as props instead?

Reply:
If this is too much of a refactor for the scope of the project, we can reconsider, but ideally, `web components' should provide language-agnostic components.

Reply:
Yes, the idea would be to send these strings as props.